packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -21,6 +22,16 @@ interface AutocompleteProps {
    */
   readonly placeholder: string;
 
+  /**
+   * Turn off checkmarks on selected result elements.
+   */
+  readonly checkmarks?: boolean;
+
+  /**
+   * Hint adjusts the interface to either have small or large spacing.
+   */
+  readonly inputSize?: FormFieldProps["size"];
```

Comment:
`size` would be more conventional here if it's controlling the size of the input.